__tests__/address.test.js

```diff
@@ -0,0 +1,48 @@
+import TestUtils from "./test-utils";
+
+describe("address api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .get("/wallet/address")
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.address).toBe("WmtWgtk5GxdcDKwjNwmXXn74nQWTPWhKfx");
```

Comment:
It should return the same address if called again, right? I think we should test it as well.

Reply:
Done